redaxo/src/core/lib/error_handler.php

```diff
@@ -51,7 +51,9 @@ public static function handleException($exception)
 
             // in case exceptions happen early - before symfony-console doRun()
             if ('cli' === PHP_SAPI) {
-                echo $exception->__toString();
+                /** @psalm-taint-escape html */
```

Comment:
Html ist hier ja inhaltlich falsch. Kann man taint auch per annotation generell abschalten?

Reply:
Ich kenne keinen anderen Weg.

Reply:
Upstream issue aufgemacht wg html sink in der CLI

https://github.com/vimeo/psalm/issues/5132